grails-app/services/au/org/ala/merit/WebService.groovy

```diff
@@ -76,11 +76,13 @@ class WebService {
             resp.error = "Timed out calling web service. URL= ${url}."
             resp.statusCode = HttpStatus.CONNECTION_TIMED_OUT
             log.warn resp.error
+        } catch (SocketException se) {
```

Comment:
This is good but probably needs to be applied in other places in the WebService as well.